src/controllers/translate.js

```diff
@@ -0,0 +1,32 @@
+import sucessRes from '../helpers/successHandler';
+import errorRes from '../helpers/errorHandler';
+import pkg from '@google-cloud/translate';
+import dotenv from 'dotenv';
+
+dotenv.config();
+const { Translate } = pkg.v2;
+
+//Google cloud translate API credentials
+const CREDENTIALS = JSON.parse(process.env.CREDENTIALS);
+
+const translate = new Translate({
+  credentials: CREDENTIALS,
+  projectId: CREDENTIALS.project_id,
+});
+
+const translateText = async (req, res) => {
+  var text = req.body.html;
+  let targetLanguage = req.body.language;
```

Comment:
could you destructure text and targetLanguage from req.body one line please? and use es6 features please